lua/telescope/builtin/__internal.lua

```diff
@@ -972,6 +972,9 @@ internal.buffers = function(opts)
       previewer = conf.grep_previewer(opts),
       sorter = conf.generic_sorter(opts),
       default_selection_index = default_selection_idx,
+      attach_mappings = function(_, map)
+        map({ "i", "n" }, "<C-x>", actions.delete_buffer)
+      end,
```

Comment:
I am not the biggest fan of mapping custom picker actions by default, thats why its currently still not mapped. Mappings are always opinionation but we can do that but should document that at least in `builtins/init.lua`, with a simple mention